### PR TITLE
Documentation Standard Updates

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
This pull request was created automatically 🎉:

- [x] Ensure `.github/` files have been properly updated (templates are updated and auto releaser is included)
- [x] Confirm all appropriate end models are included in the `.quickstart/quickstart.yml` file as `public_models` (only for transform repos)
- Verify the README has made all necessary updates:
    - [x] Removal of emojis
    - [x] Replacing exclamation points with periods in sentences
    - [x] Adjusting all headers to ensure only one H1 heading
    - [x] Confirming a section header is included for Quickstart UI rendering (only for transform repos)
    - [x] Renaming `**model` to `**table**` (only for transform repos)
    - [x] Removal of Fivetran community and office hour resources (if necessary)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,14 +14,14 @@
...
-  - Aggregates all relevant application metrics into each of the reporting levels above
+- Providing intuitive reporting at the App Version, Platform Version, Device, Source Type, Territory, Subscription and Overview levels
+- Aggregates all relevant application metrics into each of the reporting levels above
 - Generates a comprehensive data dictionary of your source and modeled Apple App Store data through the [dbt docs site](https://fivetran.github.io/dbt_apple_store/).
 
-The following table provides a detailed list of all models materialized within this package by default. 
+The following table provides a detailed list of all models materialized within this package by default.
 > TIP: See more details about these models in the package's [dbt docs site](https://fivetran.github.io/dbt_apple_store/#!/overview?g_v=1).
 
 | **model**                  | **description**                                                                                                                                               |
```
Comment (fivetran-joemarkiewicz) [inline README.md:27]:
```suggestion
| **Table**                  | **Description**                                                                                                                                               |
```

File: README.md [hunk 2]
```diff
@@ -14,14 +14,14 @@
...
 
 # Apple App Store Transformation dbt Package ([Docs](https://fivetran.github.io/dbt_apple_store/))
-# 📣 What does this dbt package do?
+## What does this dbt package do?
 - Produces modeled tables that leverage Apple App Store data from [Fivetran's connector](https://fivetran.com/docs/connectors/applications/apple-app-store) in the format described by [this ERD](https://fivetran.com/docs/connectors/applications/apple-app-store#salesandfinancereportschema) and build off the output of our [Apple App Store source package](https://github.com/fivetran/dbt_apple_store_source).
 - Enables you to better understand your Apple App Store metrics at different granularities. It achieves this by:
-  - Providing intuitive reporting at the App Version, Platform Version, Device, Source Type, Territory, Subscription and Overview levels
-  - Aggregates all relevant application metrics into each of the reporting levels above
+- Providing intuitive reporting at the App Version, Platform Version, Device, Source Type, Territory, Subscription and Overview levels
+- Aggregates all relevant application metrics into each of the reporting levels above
```
Comment (fivetran-joemarkiewicz) [inline README.md:21]:
```suggestion
  - Providing intuitive reporting at the App Version, Platform Version, Device, Source Type, Territory, Subscription and Overview levels
  - Aggregates all relevant application metrics into each of the reporting levels above
```

File: README.md [hunk 3]
```diff
@@ -34,15 +34,15 @@ The following table provides a detailed list of all models materialized within t
 | [apple_store__subscription_report](https://fivetran.github.io/dbt_apple_store/#!/model/model.apple_store.apple_store__subscription_report) | Each record represents daily subscription metrics by account, app, subscription name, country and state. |
 | [apple_store__territory_report](https://fivetran.github.io/dbt_apple_store/#!/model/model.apple_store.apple_store__source_type_report) | Each record represents daily subscription metrics by app_id, source_type and territory. |
```
Comment (fivetran-joemarkiewicz) [inline README.md:36]:
```suggestion

<!--section-end-->

```

File: README.md [hunk 4]
```diff
@@ -14,17 +14,19 @@
...
 - Produces modeled tables that leverage Apple App Store data from [Fivetran's connector](https://fivetran.com/docs/connectors/applications/apple-app-store) in the format described by [this ERD](https://fivetran.com/docs/connectors/applications/apple-app-store#salesandfinancereportschema) and build off the output of our [Apple App Store source package](https://github.com/fivetran/dbt_apple_store_source).
 - Enables you to better understand your Apple App Store metrics at different granularities. It achieves this by:
   - Providing intuitive reporting at the App Version, Platform Version, Device, Source Type, Territory, Subscription and Overview levels
   - Aggregates all relevant application metrics into each of the reporting levels above
 - Generates a comprehensive data dictionary of your source and modeled Apple App Store data through the [dbt docs site](https://fivetran.github.io/dbt_apple_store/).
 
-The following table provides a detailed list of all models materialized within this package by default. 
+<!--section=“apple_store_transformation_model"-->
+
+The following table provides a detailed list of all models materialized within this package by default.
```
Comment (fivetran-joemarkiewicz) [inline README.md:26]:
```suggestion
The following table provides a detailed list of all tables materialized within this package by default.
```

**Review by fivetran-avinash — APPROVED** [review]
One small update but this is good to go now!

**Pull request merged**